clang/test/Driver/cxx-static-destructors.cpp

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang -### -c -fc++-static-destructors=all %s 2>&1 | FileCheck --check-prefix ALL %s
```

Comment:
thx for adding a driver test. The previous `-fc++-static-destructors` patch  https://reviews.llvm.org/D50994 did not come with a driver test